test/test_untilTag.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs');
```

Comment:
Have a look at how [chai] is used [in other tests](https://github.com/dcmjs-org/dcmjs/blob/master/test/test_data.js#L1) with the `expect` method so that the individual steps of the process can be verified.

Reply:
The test has been integrated with the expect method